src/Shell/HelpShell.php

```diff
@@ -85,6 +85,32 @@ public function main()
     }
 
     /**
+     * Get the list of commands using the CommandTask
+     *
+     * Provides backwards compatibility when an application doesn't use
+     * CommandRunner.
+     *
+     * @return array
+     */
+    protected function getCommands()
+    {
+        $task = new CommandTask($this->getIo());
+        $nested = $task->getShellList();
+        $out = [];
+        foreach ($nested as $section => $commands) {
+            $prefix = '';
+            if ($section !== 'CORE' && $section !== 'app') {
```

Comment:
Not without possibly causing other issues with other consumers of the CommandTask.